src/components/Auction/Auction.jsx

```diff
@@ -181,75 +223,127 @@ class Auction extends Component {
               </div>
             </div>
+            <Description />
           </div>
-          <section className="summary">
-            <h3>Summary</h3>
-            <ul className="tabs">
-              {TABS.map((tab, index) => (
-                <li key={tab.name}>
-                  <button
-                    className={`tab__button ${index === activeTabIndex ? 'active__tab' : null}`}
-                    onClick={() => this.onTabClick(index)}
+          {currentAuctionSummary && (
+            <>
+              <section className="summary">
+                <h3>Summary</h3>
+                <ul className="tabs">
+                  {TABS.map((tab, index) => (
+                    <li key={tab.name}>
+                      <button
+                        className={`tab__button ${index === activeTabIndex ? 'active__tab' : null}`}
+                        onClick={() => this.onTabClick(index)}
+                      >
+                        <p>{tab.name}</p>
+                        <span>
+                          {this.props[tab.title] && this.renderTabTileAmount(tab.title)}
+                        </span>
+                        <span className="little__text">{this.props[tab.title] && this.renderTabPrice(tab.title)}</span>
+                      </button>
+                    </li>
+                  ))}
+                </ul>
+                <ActiveTabComponent
+                  tabContent={TABS[activeTabIndex]}
+                  data={this.props[TABS[activeTabIndex].title]}
+                />
+              </section>
+
+              <section>
+                <h3 className="current-auction__header">Current Auction - live</h3>
+                <div className="current-auction">
+                  <div className="current-auction__card">
+                    <span className="current-auction__title">
+                      Current Auction <br /> NEC Price
+                    </span>
+                    <span className="current-auction__value">
+                      {currentAuctionSummary.currentNecPrice} <small>ETH</small>
+                    </span>
+                  </div>
+                  <div className="current-auction__card">
+                    <span className="current-auction__title">
+                      Next Auction <br /> NEC Price
+                    </span>
+                    <span className="current-auction__value">
+                      {currentAuctionSummary.nextNecPrice} <small>ETH</small>
+                    </span>
+                  </div>
+                </div>
+                <div className="graphics__container">
+                  <Circle
+                   percentage="50"
+                   title="Next price change"
+                  >
+                    <text x="18" y="20" className="chart-title">
+                      {this.format(nextPriceChange)}
+                    </text>
+                  </Circle>
+                  <Circle
+                    percentage="80"
```

Comment:
Percentage should not be hard coded but should instead be adjusted as fraction of remaining/initial